File: ARC01_99/ARC51/b.py
def gcd(a, b, cnt):
    if (b == 0): return cnt
    cnt += 1
    return gcd(b, a % b, cnt)


# ユークリッドを遡ればいい
K = int(input())

cnt = 0
a, b, c = 0, 1, 0
while (cnt != K):
    a = b + c
    b, c = a, b
    cnt += 1
a = b + c
print(a, b) if(K != 1) else print(1, 1)

File: ARC01_99/ARC51/b.cpp
#include <iostream>
using namespace std;
using ll = long long;
int main() {
  ll a = 0, b = 1, c = 0;
  int K;
  cin >> K;
  int cnt = 0;
  while (cnt != K) {
    a = b + c;
    c = b;
    b = a;
    cnt++;
  }
  a = b + c;
  if (K != 1)
    cout << a << " " << b << endl;
  else
    cout << 1 << " " << 1 << endl;
  return 0;
}